lib/notiffany/notifier/tmux/client.rb

```diff
@@ -11,7 +11,7 @@ class << self
           def version
             begin
               Float(_capture("-V")[/\d+\.\d+/])
-            rescue TypeError
+            rescue NoMethodError, TypeError
```

Comment:
`TypeError` is for when `_capture` returns some string, but not containing float-like string.